components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/internal/OAuthComponentServiceHolder.java

```diff
@@ -47,6 +48,28 @@
     private List<TokenBindingMetaDataDTO> tokenBindingMetaDataDTOs = new ArrayList<>();
     private OAuthAdminServiceImpl oAuthAdminService;
 
+    public List<ScopeValidator> getScopeValidators() {
```

Comment:
Add method comments to the public methods

Reply:
added in 79d44c4e3bce8a5f1c57e5dce757c1bb19337031